x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_overview/components/stream_chart_panel.tsx

```diff
@@ -141,7 +141,7 @@ export function StreamChartPanel({ definition }: StreamChartPanelProps) {
   };
 
   const docCount = docCountFetch?.value?.details.count;
-  const formattedDocCount = docCount ? formatNumber(docCount, 'decimal0') : '-';
+  const formattedDocCount = docCount ? formatNumber(docCount, 'decimal0') : '0';
```

Comment:
The outcome is the same if we just remove the conditional and always call `formatNumber(docCount, 'decimal0')` but maybe it's safer this way